app/routes/events/components/StripeElement.js

```diff
@@ -4,38 +4,146 @@ import {
   injectStripe,
   PaymentRequestButtonElement,
   Elements,
-  StripeProvider
+  StripeProvider,
+  CardCVCElement,
+  CardExpiryElement,
+  CardNumberElement
 } from 'react-stripe-elements';
 import config from 'app/config';
-import StripeCheckout from 'react-stripe-checkout';
-import Button from 'app/components/Button';
-import styles from './StripeElement.css';
-import logoImage from 'app/assets/kule.png';
+import stripeStyles from './Stripe.css';
 import type { EventRegistrationChargeStatus, User, Event } from 'app/models';
+import LoadingIndicator from 'app/components/LoadingIndicator';
 
-import { paymentPending } from '../utils';
+type Stripe = {
+  paymentRequest: Object => Object,
+  handleCardPayment: string => Promise<*>,
+  confirmPaymentIntent: (string, Object) => Promise<*>
+};
 
 type Props = {
   event: Event,
   currentUser: User,
-  onToken: (token: string) => Promise<*>,
+  createPaymentIntent: () => Promise<*>,
   chargeStatus: EventRegistrationChargeStatus
 };
 
-type FormProps = Props & { stripe: { paymentRequest: Object => Object } };
+type StripeError = {
```

Comment:
Would be awesome with some references to where these are found

Reply:
Will do!